core/market/decentralized/src/db/dao/agreement.rs

```diff
@@ -92,6 +101,53 @@ impl<'c> AgreementDao<'c> {
         })
         .await
     }
+
+    pub async fn clean(&self) -> DbResult<()> {
+        // FIXME use grace time from config file when #460 is merged
+        log::debug!("Clean market agreements: start");
+        let interval_days = std::env::var(AGREEMENT_CONFIG.name)
+            .and_then(|v| v.parse::<u64>().map_err(|_| std::env::VarError::NotPresent))
+            .unwrap_or(AGREEMENT_CONFIG.default)
+            .max(AGREEMENT_CONFIG.min);
```

Comment:
Since we have EnvConfig struct, it should have generic implementation, that does all this stuff. It would be much more comfortable to use.

The second thing: it is important, that configuration comes from outside of market. If we get env vars here, it is semi-hardcoded. We can't change it other way than setting environment variable, what is cumbersome in some cases (like for example tests).
Check here how config can be used: https://github.com/golemfactory/yagna/blob/ece76f0d5c2187c6f784986ffc667399bc18859c/core/market/decentralized/tests/test_startup.rs#L13-L26
We can't do this, if we explicit get environment in market code.
But we can address this issue later.